src/Model/benefit.js

```diff
@@ -0,0 +1,79 @@
+class Benefit {
+	getBenefit(visitDate, menu, beforeDiscountPrice) {
+		if (beforeDiscountPrice < 10000)
+			return {
+				DdayMoney: 0,
+				weekdayOrweekendMoney: 0,
+				starMoney: 0,
+				totalBenefit: 0,
+				dateIndex: 0,
+			}
+		const dateIndex = this.#getDayOfWeek(visitDate)
+		const DdayMoney = this.#getD_Day(visitDate)
+
+		const weekdayOrweekendMoney =
+			dateIndex <= 4
+				? this.#getWeekdayBenefit(menu)
+				: this.#getWeekendBenefit(menu) //4이상이면 주말
+
+		const starMoney = this.#getStarBenefit(dateIndex, visitDate)
+
+		const totalBenefit = this.#getTotalBenefit(
+			DdayMoney,
+			weekdayOrweekendMoney,
+			starMoney,
+		)
+		return {
+			DdayMoney,
+			weekdayOrweekendMoney,
+			starMoney,
+			totalBenefit,
+			dateIndex,
+		}
+	}
+
+	getPresent(beforeDisCountPrice) {
+		return beforeDisCountPrice >= 120000 ? true : false
+	}
```

Comment:
```suggestion
	hasPresent(beforeDisCountPrice) {
		return beforeDisCountPrice >= 120000;
	}
```
이렇게 축약할 수 있을 것 같고, 보통 `Boolean`값을 리턴할 때는 `is`, `has`, `should`와 같은 접두사를 많이 사용하는 것 같아요!